internal/configs/node/network.go

```diff
@@ -77,6 +77,12 @@ const (
 	// When the peer count exceeds the 'high watermark', as many peers will be pruned (and
 	// their connections terminated) until 'low watermark' peers remain.
 	DefaultConnManagerHighWatermark = 192
+	// DefaultResourceMgrEnabled enables P2P host Resource Manager
+	DefaultResourceMgrEnabled = false
```

Comment:
here we have false because this settings are for the harmony binary, right?

Reply:
True, it is disabled by default for harmony node. For boot node, it's enabled by default.